composer.json

```diff
@@ -20,7 +20,7 @@
         "predis/predis": "~1.0",
         "twig/twig": "~1.23",
         "monolog/monolog": "~1.17",
-        "symfony/serializer": "~2.3|3.0.*",
+        "symfony/serializer": "~2.3|~3.0",
```

Comment:
How about using the Caret operator instead? Has the same meaning in this case, but would be in line how [others are using Symfony components](https://github.com/silexphp/Silex/blob/74d9869d444bcb7b0b2450de43832f4a6c894377/composer.json#L26).

`"symfony/serializer": "~2.3|^3.0"`